pootle/static/js/editor/app.js

```diff
@@ -2312,4 +2338,40 @@ PTL.editor = {
     return true;
   },
 
+  toggleSuggestion(e, canHide = false) {
+    e.stopPropagation();
+    const $suggestion = $(e.target).parents('.js-user-suggestion');
```

Comment:
`const $suggestion = $(e.target).parents('.js-user-suggestion');` builds a jQuery object whereas ``const suggestion = document.getElementById(`suggestion-${e.dataset.suggId}`);`` retrieves a DOM node. The set of APIs available on the former and latter are different, that's why the method calls need to be slightly adjusted.